src/utils/string.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Converts the provided string value that may contain the division operation to the number value.
```

Comment:
```suggestion
* Converts the provided string that may contain a division operation to a number.
```

Reply:
Thanks, I have just updated the comment for the function. By the way, I added you to the related PR for a "httpstreaming" repo (https://github.com/videojs/http-streaming/pull/1289).